applications/tari_base_node/src/bootstrap/wallet.rs

```diff
@@ -162,13 +168,22 @@ impl WalletBootstrapper {
                     broadcast_send_timeout: config.transaction_broadcast_send_timeout,
                     ..Default::default()
                 },
-                peer_message_subscriptions,
+                peer_message_subscriptions.clone(),
                 transaction_db,
                 self.node_identity.clone(),
                 self.factories,
             ))
+            .add_initializer(BaseNodeServiceInitializer::new(
+                BaseNodeServiceConfig::default(),
+                peer_message_subscriptions,
+                WalletDatabase::new(WalletMemoryDatabase::new()),
```

Comment:
Is Memory correct here?

Reply:
Yes, In the current BaseNode we dont run a full wallet, only some services. We dont currently boot up a new sql db for the private key, tor key etc. 

The normal console wallet etc does have a DB, so we can pass that in this service. This is more temp code until we remove the basenode wallet. But it also does not need persistant storage for the base_node info, as the wallet inside of the base_node will always be connected to the base_node itself.